src/include/job.h

```diff
@@ -1058,14 +1058,20 @@ task_find	(job		*pjob,
 #define JOB_EXEC_HOOK_DELETE   -19 /* a hook requested for job to be deleted */
 #define JOB_EXEC_RERUN_MS_FAIL -20 /* Mother superior connection failed */
 #define JOB_EXEC_FAIL_SECURITY -21 /* Security breach in PBS directory */
-#define JOB_EXEC_HOOKERROR	-22 /* job exec failed due to */
-				    /* unexpected exception or */
-				    /* hook execution timed out */
+#define JOB_EXEC_HOOKERROR	-22 /* job exec failed due to
+				     * unexpected exception or
+				     * hook execution timed out
+				     */
+#define JOB_EXEC_UPDATE_ALPS_RESV_ID 1 /* Update ALPS reservation ID to parent mom as soon as it is available.
+					* This is neither a success nor a failure exit code,
+					* so we are using a positive value
+					*/
```

Comment:
The first line is too long in this comment, and the rest of the lines need to be indented one space more.

Reply:
The indentation looks off in the diff. But it appears correct if you view the file. I don't know why it looks this way in this diff snippet. 
Also, fixed the comment. Squashed commits and rebased.